monero-outreach-docs/translations/fr/archive/triptych_fr.md

```diff
@@ -0,0 +1,43 @@
+# Triptych: Un Nouvel Algorithme Protégeant les Données des Utilisateurs de Monero
+
+**“Monero obfusque l'émetteur, le destinataire, et le montant des transactions. Triptych est une amélioration pour Monero qui rend son utilisation plus rapide, plus légère et plus privée.” - 1e septembre, 2020**
+
+Monero est conçu pour obfusquer les valeurs (émetteur, destinataire et montants) de chaque transaction avec des techniques et des technologies évolutives. Il s’agit de présenter ici Triptych, une technologie innovante permettant l’optimisation de la fonction d'obfuscation de l’émetteur, celui de l’usage de la chaine de blocs et une rapidité d’exécution supérieure.
+
+### _Les Signatures de Cercle_
+
+Pour assurer la confidentialité des émetteurs, Monero fait plus que garder le secret des adresses. L’ensemble de la base des données de Monero existe sur sa chaine de blocs sous forme de transactions sortantes dénommées TXO. À chaque TXO unique est assigné une identité unique (ID), appelée clé publique (également nommée "adresse furtive"). Par simple analyse de la chaine de blocs de Monero, il est impossible d'établir une corrélation entre les clés publiques des TXO et les adresses des portefeuilles électroniques de Monero. Cela étant, les clés publiques ont également besoin d’une protection. Il serait inacceptable que les flux de Monero parmi les clés publiques soient transparents, considérant que des techniques d'analyses externes pourraient associer ces clés publiques a des portefeuilles, puis à des individus.
+
+Les flux de Monero des clés publiques sont gardés confidentiels par l’usage d’un procédé de chiffrage nommé "cryptographic ring signatures". Ce procédé permet de dissimuler la clé publique d’un TXO en l'incorporant à un groupe de transaction-leurres (également nommée "mixins"). L’observation d’une transaction ne permettra donc pas de déterminer si un TXO donné est l’original ou non. Vous pouvez voir un exemple d’émission ou de réception de TXO en cherchant une transaction a l’aide d’un programme d’analyse de la chaine de blocs de Monero tel que [moneroblocks.info](https://moneroblocks.info/) ou [xmrchain.net](https://xmrchain.net/).
```

Comment:
Les flux de Monero des clés publiques sont gardés confidentiels par l’usage d’un procédé de chiffrage nommé "cryptographic ring signatures".  >>> En l'état, les flux de Monero des clés publiques sont gardés confidentiels par l’usage d’un procédé de chiffrage nommé "cryptographic ring signatures".   

Reason: Missing the translation of 'right now'

Reply:
Integrated